crates/rspack_core/src/exports_info.rs

```diff
@@ -1884,3 +1884,243 @@ macro_rules! debug_exports_info {
     }
   };
 }
+
+pub trait ModuleGraphAccessor<'a> {
+  fn run_resolve_filter(
+    &mut self,
+    target: &ResolvedExportInfoTarget,
+    resolve_filter: &ResolveFilterFnTy,
+  ) -> bool;
+  fn get_export_info(&mut self, export_info_id: &ExportInfoId) -> Arc<ExportInfo>;
+  fn get_export_info_id(
+    &mut self,
+    name: &Atom,
+    module_identifier: &ModuleIdentifier,
+  ) -> Arc<ExportInfoId>;
```

Comment:
I don't see any necessary to return a `Arc` here， since `ExportInfoId` is `Copyed`